### PR TITLE
✨(frontend) team description

### PR DESCRIPTION
**Pull request by AntoLC**
## Purpose

Create the box team description.

## Proposal

- [x] Integrate the design from the mockup

## Demo

[scrnli_2_13_2024_1-51-54 PM.webm](https://github.com/numerique-gouv/people/assets/25994652/d11a6e2e-4fd3-4f32-a782-78c938ad358e)

### CODE REVIEW

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          <Text as="h3" $weight="bold" $size="1.25rem" className="mt-0">
+            {t('Members of “{{teamName}}“', {
+              teamName: team.name,
+            })}
+          </Text>
+          <Text>
+            {t('Add people to the “{{teamName}}“ group.', {
+              teamName: team.name,
+            })}
+          </Text>
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:39]: nit: font-size is of 13px on the design file !

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  team: TeamResponse;
+}
+
+export const TeamInfo = ({ team }: TeamInfoProps) => {
+  const { t } = useTranslation();
+  const { colorsTokens } = useCunninghamTheme();
+
+  return (
+    <Card className="m-b" style={{ paddingBottom: 0 }}>
+      <Box className="m-b" $direction="row" $align="center" $gap="1.5rem">
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:20]: The icon seems to be aligned on the top on the figma design, thus, not moving when resizing the window

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            })}
+          </Text>
+        </Box>
+      </Box>
+      <Box
+        className="p-s"
+        $gap="1rem"
+        $direction="row"
+        $justify="space-evenly"
+        $css="border-top: 1px solid #e3e3e3;"
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:47]: nit: johann seems to have used a slightly different gray variation for border, `#DDDDD` do we ignore it?

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+interface TeamInfoProps {
+  team: TeamResponse;
+}
+
+export const TeamInfo = ({ team }: TeamInfoProps) => {
+  const { t } = useTranslation();
+  const { colorsTokens } = useCunninghamTheme();
+
+  return (
+    <Card className="m-b" style={{ paddingBottom: 0 }}>
```
[the remark this answers is not in the document]
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:19]: If we like it so. It is to be lightly more card way.

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx (hunk 1 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx]: [64aa8ad](https://github.com/numerique-gouv/people/pull/49/commits/64aa8adebdfb1b797a4b66af063a5a1c026ddf92)

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx (hunk 2 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:20]: [64aa8ad](https://github.com/numerique-gouv/people/pull/49/commits/64aa8adebdfb1b797a4b66af063a5a1c026ddf92)

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      >
+        <Text $size="s">
+          {t('{{count}} member', { count: team.accesses.length })}
+        </Text>
+        <Text $size="s">
+          {t('Created at {{created_at}}', { created_at: '06/02/2024' })}
+        </Text>
+        <Text $size="s">
+          {t('Last update at {{updated_at}}', { updated_at: '07/02/2024' })}
+        </Text>
```
[the remark this answers is not in the document]
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx]: [64aa8ad](https://github.com/numerique-gouv/people/pull/49/commits/64aa8adebdfb1b797a4b66af063a5a1c026ddf92)

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx (hunk 3 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx]: [64aa8ad](https://github.com/numerique-gouv/people/pull/49/commits/64aa8adebdfb1b797a4b66af063a5a1c026ddf92)

File: src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (lebaudantoine) [reply src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx:19]: okay, let's keep it that way. At some point, the product team will review the UI.

**Review by lebaudantoine — APPROVED** [review]

**Pull request merged**